src/exe/colmap.cc

```diff
@@ -226,6 +226,49 @@ int RunColorExtractor(int argc, char** argv) {
   return EXIT_SUCCESS;
 }
 
+int RunDatabaseCleaner(int argc, char** argv) {
+  int cleanup_type = -1;
+
+  OptionManager options;
+  options.AddRequiredOption("cleanup_type", &cleanup_type,
+                            "{0: All, 1: Images, 2: Features, 3: Matches}");
```

Comment:
It might be more natural to expose these options as strings, such that I could write `colmap database_cleaner --database_path path/to/db --type all` instead of `--type 0`.

Reply:
Not sure why I used numeric values in the first place. I'll update them to strings.